packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -25,6 +25,9 @@ import {
   getCaretDownButtonStyles
 } from './ComboBox.styles';
```

Comment:
Needs a change file. Can you please run `npm run change -- -b upstream/master` from the root?

Reply:
@dzearing , change file created, let me know if there's anything else :)